src/components/TabNavigation/tabNavigation.js

```diff
@@ -1,8 +1,6 @@
 import React from "react"
 
-import pageGroup from "../../data/pageGroups.yml"
```

Comment:
Was able to use the new data provider instead of having to import this. Had to touch up the data to get the necessary props.